migration_steps/prepare_source_data/counts_verification/sql/count_casrec_source.sql

```diff
@@ -463,6 +463,42 @@ UPDATE countverification.counts SET {working_column} =
 )
 WHERE supervision_table = 'person_warning';
 
+-- annual_report_letter_status
+UPDATE countverification.counts
+SET {working_column} = 0
+WHERE supervision_table = 'annual_report_letter_status';
+
+-- annual_report_type_assignments
```

Comment:
NB work in IN-1168 is likely to make this incorrect, as we are now deriving annual report type assignments from PENDING reports only. Perhaps you should merge this and I'll deal with the consequences in my branch? I think it's only this query which will need to change; if I'm understanding correctly, the other queries will still be correct as they use this one as their source data?

Reply:
This is potentially going to be a bit convoluted, as the annual_report_type_assignments are now derived indirectly from casrec, from the annual_report_logs table.